src/main/webapp/app/overview/overview.component.html

```diff
@@ -28,6 +28,23 @@ <h3>
     <div class="col-12 col-sm">
         <h3 jhiTranslate="artemisApp.studentDashboard.title">Your current courses</h3>
     </div>
+    <div *jhiHasAnyAuthority="['ROLE_ADMIN']">
+        <div class="btn btn-primary" *ngIf="!startLoginProcess && !selectedUserLogin" (click)="startLoginProcess = !startLoginProcess">
+            {{ 'artemisApp.studentDashboard.studentLogin.loginAsStudent' | translate }}
+        </div>
+        <div *ngIf="!startLoginProcess && selectedUserLogin">
+            {{ 'artemisApp.studentDashboard.studentLogin.loggedInAs' | translate }} {{ selectedUserLogin }}
+            <div class="btn btn-primary" (click)="removeUserLogin()">
+                {{ 'artemisApp.studentDashboard.studentLogin.logout' | translate }}
+            </div>
+        </div>
+        <div class="input-group" *ngIf="startLoginProcess">
+            <input [(ngModel)]="selectedUserLogin" />
```

Comment:
The implementation seems barebones to me:
- No validation if the input user name is correct (you can just input anything and press the login button)
- No autocomplete (even though this might be as intended by @krusche )
- The Login button is not disabled if the `selectedUserLogin` is empty or invalid